openmeetings-service/src/main/java/org/apache/openmeetings/service/mail/template/FeedbackTemplate.html

```diff
@@ -20,12 +20,23 @@
 -->
 <!DOCTYPE html>
 <html xmlns:wicket="http://wicket.apache.org/dtds.data/wicket-9.xsd">
-<wicket:extend>
-	<strong><span wicket:id="appname"></span> - Feedback</strong><br />
-	Message from User: <span wicket:id="username"></span><br/>
-	Mail: <span wicket:id="email"></span><br/>
-	Message: <span wicket:id="message"></span><br/><br/>
-	<strong>Click on this link to participate in the Meeting:</strong><br/>
-	<a wicket:id="invitation_link">Click here to enter room</a><br/>
-</wicket:extend>
+<div class="main">
```

Comment:
one quick comment:

```
<div class="main">
    <div class="container">
      <div class="wrapper">
```

being repeated in all `*.html` files in such case it should be moved to `AbstractTemplatePage.html` as folloows:

```
	<body class="main">
		<div wicket:id="container" class="container">
			<div class="wrapper">
				<wicket:child/>
			</div>
		</div>
	</body>
```

is it necessary to have both `<div class="container">` and `<div class="wrapper">` ? IMO we can have only one of them :)

please NOTE indentation should be made with tabs, not spaces :)

Reply:
Hi, Ok, I'll move it to `AbstractTemplatePage.html` to do not repeat my self :)
And yes, we can have one of them, I will fix the style and remove one `<div></div>`.

Reply:
Hello, 
- fixed the style to have the `<body></body>` with its styles, 
- updated the `<div class="main"></div>` to center the `<div class="wrapper"></div>` with specific padding and 
- removed the `<div class="container"></div>`.